gen3testing.braincommons.org/etlMapping.yaml

```diff
@@ -1,5 +1,5 @@
 mappings:
-  - name: bhcqa_etl
+  - name: bhcdatastaging_etl
```

Comment:
are we calling these indices `bhcdatastaging_*`? we were calling them `bhcqa_*`

Reply:
good catch. I updated